src/librustc_middle/ty/context.rs

```diff
@@ -2701,7 +2701,8 @@ pub fn provide(providers: &mut ty::query::Providers<'_>) {
         let id = tcx.hir().local_def_id_to_hir_id(id.expect_local());
         tcx.stability().local_deprecation_entry(id)
     };
-    providers.extern_mod_stmt_cnum = |tcx, id| tcx.extern_crate_map.get(&id).cloned();
+    providers.extern_mod_stmt_cnum =
+        |tcx, id| tcx.extern_crate_map.get(&id.expect_local()).cloned();
```

Comment:
Looks like this query can use `LocalDefId` as an input.

Reply:
Done